scripts/ec2_setup.sh

```diff
@@ -0,0 +1,57 @@
+#!/usr/bin/env bash
+
+set -e
+set -x
+
+#
+# Dependencies
+#
+
+# Packages
+sudo yum groupinstall -y "Development Tools"
```

Comment:
This implies that the EC2 instance is Amazon Linux (or RHEL or a derivative). Running such script on a Debian/Ubuntu EC2 instance will certainly fail with a `"Command not found: yum"` type error. May be worth adding a comment in the script, e.g.
```
#
# Dependencies (assuming `yum` as a package manager)
#
```

Reply:
added some comments and a check in the script